survey-sdk/src/main/java/co/nimblehq/survey/sdk/ResultSdk.kt

```diff
@@ -1,13 +1,13 @@
-package co.nimblehq.sample.data
+package co.nimblehq.survey.sdk
 
 /**
  * A generic class that holds a value with its loading status.
  * @param <T>
  */
-sealed class Result<out T : Any> {
+sealed class ResultSdk<out T : Any> {
 
-    data class Success<out T : Any>(val data: T) : Result<T>()
-    data class Error(val exception: Exception) : Result<Nothing>()
+    data class Success<out T : Any>(val data: T) : ResultSdk<T>()
+    data class Error(val exception: Exception) : ResultSdk<Nothing>()
```

Comment:
this name is kinda weird. `Sdk` means the whole package of `Software Development Kit` - why would we want to name it this way? 🤔  I think result should be good enough, remember that you already had the namespace above:

`co.nimblehq.survey.sdk.Result` - this should be enough for the differentiation purpose